src/renderers/shared/stack/reconciler/__tests__/ReactCompositeComponentState-test.js

```diff
@@ -140,8 +140,7 @@ describe('ReactCompositeComponent-state', function() {
     );
     ReactDOM.render(
       <TestComponent stateListener={stateListener} nextColor="green" />,
-      container,
-      instance.peekAtCallback('setProps')
```

Comment:
I don't think we want to remove this one.  This isn't actually the same as the setProps that you removed elsewhere.